### PR TITLE
Graph list ordering

### LINKED ISSUE #234: Lists in lineage graph are not in alphabetical order
### Describe the bug
In the lineage graph the lists for resources, packages and tags are not in alphabetical order. While not a major issue this becomes an annoyance with large projects, especially the tags as we use these a lot.

### Steps To Reproduce
Add tags to a few models and view the lineage graph. In the "tags" drop down list the tags will not be in order. Same applies to the "resources" and "packages" drop down lists.
image

### Expected behavior
Lists are in alphabetical order.

### Screenshots and log output
Not applicable.

**The output of `dbt --version`:**
```
installed version: 0.21.1
   latest version: 1.0.0

Your version of dbt is out of date! You can find instructions for upgrading here:
https://docs.getdbt.com/docs/installation

Plugins:
  - postgres: 0.21.1
  - redshift: 0.21.1
  - bigquery: 0.21.1
  - snowflake: 0.21.1
```

### PR DESCRIPTION
**Pull request by pgoslatara**
resolves #234

### Description

In the drop down lists in the lineage graph, ensure the items are in alphabetical order.

### Checklist
 - [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [x] I have generated docs locally, and this change appears to resolve the stated issue
 - [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt next" section.

### CODE REVIEW

File: src/app/services/node_selection_service.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ angular
         options: {
             packages: [],
             tags: [null],
-            resource_types: ['model', 'seed', 'snapshot', 'source', 'test', 'analysis', 'exposure', 'metric'],
+            resource_types: ['model', 'seed', 'snapshot', 'source', 'test', 'analysis', 'exposure', 'metric'].sort(),
```
Comment (emmyoop) [inline src/app/services/node_selection_service.js:39]: Let's take the sort off of this one.  This order is purposeful in the what users are generally looking for.  Most of the time it's a `model` and having it at the top prevents having top hunt down the word.

**Review by emmyoop — CHANGES_REQUESTED** [review]
@pgoslatara thanks so much for putting this together.  

Sorry it's taken a bit to get around to reviewing this.  I have one small change request and then you'll also need to generate a changelog entry.  If you pull in the latest changes to the repo you can just add your changelog under the 1.1.0 heading that exists there.

File: src/app/services/node_selection_service.js (hunk 1 above)
Comment (pgoslatara) [reply src/app/services/node_selection_service.js]: Resolved: 5be3802bf59826ea0ae931c643990c16e50ad288.

I appreciate there is some logic here and that most of the time people (including me!) are looking for the "models" item however this is the only place I know of where the order is not alphabetical and that's a bit of an unexpected inconvenience.

**Review by emmyoop — APPROVED** [review]
This looks great!  Thanks so much for your contribution!

**Pull request merged**